examples/api/test/material/material_state/material_state_border_side.0_test.dart

```diff
@@ -0,0 +1,69 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/material_state/material_state_border_side.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets(
+    'MaterialStateBorderSideExampleApp displays FilterChip',
```

Comment:
This test isn't necessary. "await tester.tap(find.byType(FilterChip));" already finds exactly one chip in other test.

Reply:
Sure I removed it in [test: Remove duplicated test](https://github.com/flutter/flutter/pull/151089/commits/0e15c94eb589a8e246b4eb6ce70cc9160c2637e1)